flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/utils/TableTestBase.scala

```diff
@@ -253,18 +263,121 @@ abstract class TableTestUtilBase(test: TableTestBase, isStreamingMode: Boolean)
       }
       val types = fieldTypes.map(TypeConversions.fromLegacyInfoToDataType)
       val names = FieldInfoUtils.getFieldNames(typeInfo)
-      TableSchema.builder().fields(names, types).build()
+      TableSchema.builder().fields(names, types).build().toSchema
```

Comment:
Why here still use the deprecated TableSchema instead of `Schema.newBuilder().fromFields(names, types).build()`?

Reply:
To reduce the workload, I simply added a toSchema function. How about postponing this work until removing TableSchema?